### PR TITLE
Blog App: Views

### PR DESCRIPTION
**Pull request by Santiago220991**
# Tasks done:

- Added partials to reuse.
- Created styling files: users.css and posts.css.
- Added styling to user's views.
- Added styling to post views.

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@ class User < ApplicationRecord
   has_many :posts
   validates :name, presence: true
   validates :posts_counter, numericality: { greater_than_or_equal_to: 0, only_integer: true }
-  def last_posts(id)
+  def last_posts
     Post.where(author: id).order(created_at: :desc).limit(3)
   end
```
Comment (Davidosky007) [inline app/models/user.rb:9]: * Kindly work on these requirements, your app should function as the wireframe given below ⬇️ 
* Kindly make sure your app can handle these URLs below even after adding users, posts and comments using the rails console 
image
image
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @Santiago220991 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
## Highlights
* No linter errors 👍 
* Well writen PR description 👍 
* Good looking readme file 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/models/user.rb (hunk 1 above)
Comment (Santiago220991) [reply app/models/user.rb:9]: Hi @Davidosky007. I checked the project and It's working. I think the problem is that you are testing without adding elements to the database. I think is a good idea to try adding some elements using the rails console like this: 

first_user = User.create!(name: 'Tom', photo: 'https://images.unsplash.com/photo-1508921912186-1d1a45ebb3c1?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8', bio: 'Teacher from Mexico.', post_counter:0)
second_user = User.create!(name: 'Lilly', photo: 'https://images.unsplash.com/photo-1508921912186-1d1a45ebb3c1?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8', bio: 'Teacher from Poland.', posts_counter:0)
first_post = Post.create!(author: first_user, title: 'Hello', text: 'This is my first post', comments_counter: 0, likes_counter: 0)
second_post = Post.create!(author: first_user, title: 'Hello', text: 'This is my second post', comments_counter: 0, likes_counter: 0)
third_post = Post.create!(author: first_user, title: 'Hello', text: 'This is my third post', comments_counter: 0, likes_counter: 0)
fourth_post = Post.create!(author: first_user, title: 'Hello', text: 'This is my fourth post', comments_counter: 0, likes_counter: 0)
fifth_post = Post.create!(author: first_user, title: 'Hello', text: 'Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis p', comments_counter: 0, likes_counter: 0)
Comment.create!(post: first_post, author: second_user, text: 'Hi Tom first!' )
Comment.create!(post: first_post, author: second_user, text: 'Hi Tom second!' )
Comment.create!(post: first_post, author: second_user, text: 'Hi Tom third' )
Comment.create!(post: first_post, author: second_user, text: 'Hi Tom fourth' )
Comment.create!(post: first_post, author: second_user, text: 'Hi Tom fifth' )
Comment.create!(post: first_post, author: second_user, text: 'Hi Tom sixth' )
Like.create!(post: first_post, author: second_user)

Screenshot from 2022-08-17 07-58-45
Screenshot from 2022-08-17 07-59-05
Screenshot from 2022-08-17 07-59-14
Screenshot from 2022-08-17 07-59-28

**Review by Kingstalux — APPROVED** [review]
Hi @Santiago220991,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark: 
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

## Highlights :clap: 
:ballot_box_with_check: App has all required routes :fire: 
:ballot_box_with_check: Good commit history :+1: 
:ballot_box_with_check: Linters are passing :ok_hand: 

Good job so far :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Santiago220991** [comment]
Thanks. :+1:

**Pull request merged**